public/api/v2/config

```diff
@@ -26,6 +26,7 @@
     "oauth2": "Olaf2",
     "saml": "aufSAMLn.de"
   },
+  "plantumlServer": "http://www.plantuml.com/plantuml",
```

Comment:
should this be pre-filled here? (also it's `http`, not `https`)

Reply:
This is the mock answer to the call to the api, so this is fine for now.

Reply:
https redirects to http.

Reply:
I don't think it is a good idea to use a service which fails to properly implement https and handles user data.

Reply:
@SuperSandro2000 that's why I don't set the plantumlServer by default. `public/api/v2/config ` is a mock response for the configuration API endpoint, this won't be in production.

Reply:
I don't even know if we have to include this into the backend config. The plantuml rendering is nothing where the backend is involved. Maybe we should put this into the frontend config instead?

Reply:
Btw: we don't even know if the plantuml plugin will be included and bundled into the frontend. (After we created the frontend plugin system)

Reply:
Forget the point with the frontend config. The system with the two configs could be reworked soon.

Reply:
if this is a mock response, the value should be `https://example.com/plantuml`.

Reply:
No, because then we don't know if it's working.